Makefile

```diff
@@ -350,15 +350,21 @@ tests/scenarios/%.avm-simulation.unit: tests/scenarios/%.avm-simulation
 ## AVM Symbolic Proof Tests
 ###########################
 
-avm_prove_tests := $(wildcard tests/specs/*-spec.k)
+avm_prove_tests := $(wildcard tests/specs/*-spec.k) $(wildcard tests/specs/*/*-spec.k)
 
 test-avm-semantics-prove: $(avm_prove_tests:=.prove)
```

Comment:
Do the symbolic tests pass? If yes, we should enable them in CI. I suggest adding `test-avm-semantics-prove` into `test:`.